modules/common/src/test/scala/surge/internal/health/supervisor/HealthSupervisorActorSpec.scala

```diff
@@ -51,8 +51,8 @@ class HealthSupervisorActorSpec
       bus.signalStream().start()
 
       // Register
-      val done = bus.register(probe.ref, componentName = "boomControl", Seq(Pattern.compile("boom")))
-      done shouldBe a[Future[Ack]]
+      val done = Await.result(bus.register(probe.ref, componentName = "boomControl", Seq(Pattern.compile("boom"))), 10.seconds)
```

Comment:
you could mix in the ScalaFutures trait from ScalaTest (http://doc.scalatest.org/3.0.1-2.12/org/scalatest/concurrent/ScalaFutures.html) to avoid the Await.result ... up to you though ;-)

Reply:
Great idea! Will do.